### PR TITLE
Wazo 890 integration google plugin

### PR DESCRIPTION
**Pull request by pc-m**

### CODE REVIEW

**Comment by sileht** [comment]
I rebased it.

File: wazo_dird/plugins/google_backend/tests/test_google_view.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Copyright 2019 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0+
+
+from unittest import TestCase
+from mock import Mock, ANY
+
+from ..http import GoogleList, GoogleItem
```
Comment (sileht) [inline wazo_dird/plugins/google_backend/tests/test_google_view.py]: module pass looks wrong.

File: wazo_dird/plugins/google_backend/tests/test_service.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+import unittest
+
+from hamcrest import (
+    assert_that,
+    contains,
+    contains_inanyorder,
+    has_entries,
+)
+
+from ..import services
```
Comment (sileht) [inline wazo_dird/plugins/google_backend/tests/test_service.py]: Same here.

Comment (pc-m) [reply wazo_dird/plugins/google_backend/tests/test_service.py]: this one did not change

**Comment by pc-m** [comment]
recheck

**Comment by sileht** [comment]
recheck

**Comment by pc-m** [comment]
recheck

**Comment by fredericlepied** [comment]
recheck

**Pull request merged**